templates/settings-admin.php

```diff
@@ -32,25 +32,25 @@
 	<p>
 		<label class="password-policy-number-option">
 			<input id="password-policy-min-length" type="number" value="<?php p($_['minLength']) ?>" />
-			<span><?php p($l->t('Minimal length')) ?></span>
+			<span><?php p($l->t('Minimum password length')) ?></span>
 		</label>
 	</p>
 	<p>
 		<label class="password-policy-number-option">
 			<input id="password-policy-history-size" type="number" value="<?php p($_['historySize']) ?>" />
-			<span><?php p($l->t('User password history')) ?></span>
+			<span><?php p($l->t('User passwords history')) ?></span>
```

Comment:
the singular use of "password" is actually correct here

Reply:
Corrected 😉.

Reply:
Thank you :)